### PR TITLE
Cart management

### PR DESCRIPTION
**Pull request by stutisk**
added Add to Cart 
Remove From cart
add to wishlist 
go to Wishlist  functionalities

### CODE REVIEW

File: src/Components/Cards/Cartpagecard.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { FaShoppingCart, FaHeart } from "../Icons";
+import "../../styles/Cartpage.css";
+import "../../styles/ProductListing.css";
+
+import React from "react";
+// import { useCart } from "../../Context/CartContext";
+import { HiMinusCircle, HiPlusCircle ,AiFillDelete} from "../Icons";
+const Cartpagecard = ({ product }) => {
+  // const { state, dispatch } = useCart();
+  // console.log(cart)
```
Comment (Hardiegogo) [inline src/Components/Cards/Cartpagecard.jsx]: remove unused code

File: src/Components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -1,38 +1,70 @@
...
+              Men
+            </a>
+            <a className="nav-link padding " href="">
+              Kids
+            </a>
+          </div>
+          <div className="search-bar m-5">
+            <form action="#">
+              <input
+                type="search"
```
Comment (Hardiegogo) [inline src/Components/Navbar/Navbar.jsx:29]: You can use a code formatter for better readability.

File: src/Components/ProductListing/CartPageListing.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  const [products, setProducts] = useState([]);
+  async function getItems() {
+    try {
+      const res = await axios.get("/api/products");
+      console.log(res.data);
+      setProducts(res.data.products);
+    } catch (error) {
+      console.log(error);
+    }
+  }
```
Comment (Hardiegogo) [inline src/Components/ProductListing/CartPageListing.jsx:18]: Try abstracting this function out in a different folder (Items-utility or api-utility)

File: src/Components/ProductListing/ProductListing.jsx [hunk 4]
```diff
@@ -21,51 +22,10 @@ const ProductListing = () => {
...
-              <button className="wishlist-badge input-padding ">
-                <FaHeart size={25}/>
-              </button>
-            </div>
-          </div>
-        );
-      })}
+    <div className="products-container m-2t m-l5">
+      {
+      products.map((products) => <Productcard products={products}/>)
```
Comment (Hardiegogo) [inline src/Components/ProductListing/ProductListing.jsx:27]: key should be added here

**Review by Hardiegogo — COMMENTED** [review]
I have suggested some minor changes. ✔

File: src/Components/ProductListing/ProductListing.jsx [hunk 5]
```diff
@@ -1,11 +1,12 @@
 import "../../styles/ProductListing.css";
-import React, { useState, useEffect } from "react";
+import React from "react";
+import {Productcard} from "../Cards/Productcard";
+import {useEffect ,useState} from "react";
 import axios from "axios";
-import { FaShoppingCart,FaHeart } from"../Icons";
+;
```
Comment (kavyakunder) [inline src/Components/ProductListing/ProductListing.jsx:6]: semi-colon not needed!

**Review by kavyakunder — APPROVED** [review]
You can use prettier to format your code...
Apart from that, the code looks good to me!

**Pull request merged**